chapters/de/chapter1/1.mdx

```diff
@@ -0,0 +1,52 @@
+# Einführung
+
```

Comment:
We recently added some meta content to the English version, so you'll need to add things like the following in each section:

```suggestion

<CourseFloatingBanner
    chapter={1}
    classNames="absolute z-10 right-0 top-0"
/>

```

Check out the corresponding English files for the snippet you can copy-paste :)